<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>DQ</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Nominal Value </string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="uEdit"/>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="dqCombo">
     <item>
      <property name="text">
       <string>Contains</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Equals</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Does not contain</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Does not equal</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="nEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Uncertainty</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="nCombo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>Equals</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Not equals</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Less than</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Greater than</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="uCombo">
     <item>
      <property name="text">
       <string>Equals</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Not equals</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Less than</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Greater than</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="dqEditButton">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
